components/SiteHead.tsx

```diff
@@ -0,0 +1,33 @@
+import Head from 'next/head';
+
+type Props = {
+  title?: string;
+  description?: string;
+  image?: string;
+  url?: string;
+};
+
+export default function SiteHead({ title, description, image, url }: Props) {
```

Comment:
Better set defaults in one place, see https://react.dev/learn/passing-props-to-a-component#specifying-a-default-value-for-a-prop

```suggestion
export default function SiteHead({ 
  title = 'Nouns Terminal',
  description = 'Advanced interface for Nouns Auction',
  image = '/og_image.png',
}: Props) {
```